Content.Server/GameObjects/Components/Power/PoweredLightComponent.cs

```diff
@@ -141,9 +154,9 @@ public void UpdateLight()
             switch (LightBulb.State)
             {
                 case LightBulbState.Normal:
-                    device.Load = Load;
                     if (device.Powered)
                     {
+                        device.Load = Load;
```

Comment:
I'm pretty sure this change is gonna re-introduce the bug where lights go strobe light mode when running out of power. Why are you changing it?

Reply:
Tested it, didn't know that was a thing. Reverted the change.